messages/en.js

```diff
@@ -138,16 +138,16 @@ Blockly.Msg.DATETIME_CONVERT_FROM_DATE = "get %1 from timestamp %2";
 Blockly.Msg.DATETIME_CONVERT_FROM_DATE_TOOLTIP = "Get the specified information from a timestamp";
```

Comment:
Should all the tooltips end with a period? If so, we could uniformize at least the French translation

Reply:
Seems that indeed the standard Blockly blocks (offered by Google) have a period at the end of their tooltips:

![image](https://user-images.githubusercontent.com/14224149/130400590-5a074145-8cbd-4f89-a642-e9fbf02a5539.png)

So if you like you can change that.

Reply:
I uniformized it in 34c153562da0813f8137e43720f3a91f94bdc85c